sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClientOptions.java

```diff
@@ -194,4 +195,22 @@ public IdentityClientOptions setHttpClient(HttpClient httpClient) {
         this.httpClient = httpClient;
         return this;
     }
+
+    /**
+     * Specifies the database to extract IntelliJ cached credentials from.
+     * @param keePassDatabasePath the database to extract intellij credentials from.
+     * @return IdentityClientOptions
+     */
+    public IdentityClientOptions setKeePassDatabasePath(String keePassDatabasePath) {
+        this.keePassDatabasePath = keePassDatabasePath;
+        return this;
+    }
+
+    /**
+     * Get the KeePass database path.
+     * @return the keePass database path to extract inellij credentials from.
+     */
+    public String getIntelliJKeePassDatabasePath() {
```

Comment:
Naming mismatch between getter and setter should be resolved.

Reply:
This class is not public API.
Updated.

Reply:
We still aim to have consistently named API even when it is not public! :)

Reply:
Yep, agreed.